build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
Made some progess. Running `docker run debian env` indicates it is run as root. When building a custom image from `debian` and adding a `USER` directive to the `Dockerfile` and then running `docker run custom env` it errors that it can't find the user, which indicates that it probably tries to run the command with the configured user. Very interesting.